repokitteh.star

```diff
@@ -5,8 +5,15 @@ use("github.com/repokitteh/modules/circleci.star", secret_token=get_secret('circ
 use(
   "github.com/repokitteh/modules/ownerscheck.star",
   paths=[
-    ("envoyproxy/api-shepherds!", "api/"),
-    ("envoyproxy/udpa-wg", "api/udpa/"),
+    {
+      "owner": "envoyproxy/api-shepherds!",
+      "path": "api/",
+      "label": "api",
+    },
+    {
+      "owner": "envoyproxy/udpa-wg",
+      "path": "api/udpa/",
```

Comment:
UDPA now lives in a separate repo, so you can drop this.

Reply:
👍